packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1908,6 +1908,7 @@ export default class Meeting extends StatelessWebexPlugin {
           canResumeRecording: MeetingUtil.canUserResume(payload.info.userDisplayHints),
           canRaiseHand: MeetingUtil.canUserRaiseHand(payload.info.userDisplayHints),
           canLowerAllHands: MeetingUtil.canUserLowerAllHands(payload.info.userDisplayHints),
+          canLowerSomeoneElsesHand: MeetingUtil.canUserLowerSomeoneElsesHand(payload.info.userDisplayHints),
```

Comment:
issue: There isn't a display hint provided by locus for `LOWER_SOMEONE_ELSES_HAND` - this is one that needs to be added in (i.e. a fake display hint) when parsing display hints, or finding a different way of determining if you can lower someone else's hand (i.e. if you are a host or a cohost).